tools/js_size_diff.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -euo pipefail
+
+get_bundle_info() {
+  bazel build //enterprise/app:app_bundle >&2
+  local bundle_path="bazel-bin/enterprise/app/app_bundle/app.js"
+  local size_bytes
+  local compressed_size_bytes
+  size_bytes=$(cat "$bundle_path" | wc -c)
+  compressed_size_bytes=$(cat "$bundle_path" | gzip | wc -c)
```

Comment:
The build we serve in production is built with `bazel build -c opt` which minifies the js bundle also

Reply:
Ah right -- fixed